daliuge-engine/dlg/apps/simple.py

```diff
@@ -63,6 +63,36 @@ class NullBarrierApp(BarrierAppDROP):
     def run(self):
         pass
 
+##
```

Comment:
It's not crystal clear what the idea of this new `PythonApp` is, given it's almost identical to `SleepApp` except for the name and for not having the `sleepTime` parameter. If it's here to stay let's at least add a test for it just to ensure it's actually correct (it does look correct though).

Reply:
As mentioned in the original comment for this PR the PythonApp is required to enable people to develop a graph and a palette from scratch in EAGLE. It always existed for that reason, but was part of the hard-coded palette in EAGLE. Since a PythonApp is in some way such a fundamental component in DALiuGE, there was no class with that name in the code and thus this was the only normal component missing from the EAGLE hard-coded palette. This class only exists to enable the automatic creation of the associated component in the Raw Component palette.  Since the appclass parameter in the EAGLE doxygen section is actually empty, the code will in fact never be called (although it is functional). Thus we could go a step further and just put a 'pass' into the code body.